modules/openapi-generator/src/main/resources/cpp-ue4/model-source.mustache

```diff
@@ -175,9 +175,20 @@ bool {{classname}}::FromJson(const TSharedPtr<FJsonValue>& JsonValue)
 
 	bool ParseSuccess = true;
 
-    {{#vars}}
+	{{#requiredVars}}
```

Comment:
WriteJson uses #vars and #required to perform something similar here, if possible i'd like to keep things consistent, so if it's possible to iterate on all the vars with #vars rather than #requiredVars and #optionalVars, this would put me more at ease, and be more consistent with the place those variables are declared in the first place. Alternatively, change where those variables are declared in the header, but I'd suggest the first approach.

Reply:
Implemented request change and tested.  Works fine.  Will push shortly.